test/application_system_test_case.rb

```diff
@@ -4,16 +4,52 @@
 require "capybara/rails"
 require "capybara/minitest"
 
+require "axe/matchers/be_axe_clean"
+require "axe/expectation"
+
 require "test_helpers/cuprite_setup"
 
 Ferrum::Browser.new(process_timeout: 60, timeout: 60)
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
   driven_by :cuprite, using: :chrome, screen_size: [1400, 1400]
 
+  AXE_RULES_TO_SKIP = [:region].freeze
```

Comment:
Can we add a comment for this? Reading the code, it's not apparent to me why we are skipping regions.